test/reputation-system/reputation-mining-client/client-auto-functionality.js

```diff
@@ -381,6 +382,10 @@ process.env.SOLIDITY_COVERAGE
           });
 
           await badClient.submitRootHash();
+
+          // Forward time again so clients can start responding to challenges
+          await forwardTime(MINING_CYCLE_DURATION * 0.5, this);
```

Comment:
Why not `/ 2` like everywhere else?

Reply:
T'was just how it already was in this test but I will change it.